scripts/lib/sign-win32.ts

```diff
@@ -52,8 +52,16 @@ export async function sign(workDir: string) {
 
   // Sign individual files.  See https://github.com/electron-userland/electron-builder/issues/5968
   const unpackedDir = path.join(workDir, 'unpacked');
+  const internalDir = 'resources/resources/win32/internal';
 
-  for (const subDir of ['.', 'resources/resources/win32/internal']) {
+  // make privileged-service.exe available to the instller during signing
+  const privilegedServiceFile = 'privileged-service.exe';
+  const privilegedServiceFrom = path.join(unpackedDir, internalDir, privilegedServiceFile);
+  const privilegedServiceTo = path.join(workDir, 'win-unpacked/resources/resources/win32/internal', privilegedServiceFile);
```

Comment:
The paths involved should be:
Type | Path
--- | ---
Source | `unpackedDir` + `resources/resources/win32/internal/privileged-service.exe`
Destination | `process.cwd()` + `resources/win32/internal/privileged-service.exe`